### ISSUE TITLE
Scylla Enterprise 2022 rc8 (but I'd expect 5.0 too) errors out in scylla-image-post-start.service

### ISSUE THREAD

**Issue by tarzanek** [scylladb/scylla-machine-image#375]
Use r5 instance type in AWS, try to use the AMI, you get:

```
scyllaadm@ip-172-31-31-243:~$ sudo systemctl status scylla-image-post-start.service
● scylla-image-post-start.service - Scylla cloud image post start service
     Loaded: loaded (/lib/systemd/system/scylla-image-post-start.service; enabled; vendor preset: enabled)
     Active: failed (Result: exit-code) since Wed 2022-07-13 14:39:42 UTC; 12min ago
    Process: 7843 ExecStartPre=/bin/sleep 30 (code=exited, status=0/SUCCESS)
    Process: 7853 ExecStart=/opt/scylladb/scylla-machine-image/scylla_post_start.py (code=exited, status=1/FAILURE)
   Main PID: 7853 (code=exited, status=1/FAILURE)

Jul 13 14:39:42 ip-172-31-31-243 scylla_post_start.py[7853]: Error getting user data: HTTP Error 404: Not Found
Jul 13 14:39:42 ip-172-31-31-243 scylla_post_start.py[7853]: Traceback (most recent call last):
Jul 13 14:39:42 ip-172-31-31-243 scylla_post_start.py[7853]:   File "/opt/scylladb/scylla-machine-image/libexec/scylla_post_start.py", line 70, in <module>
Jul 13 14:39:42 ip-172-31-31-243 scylla_post_start.py[7853]:     if init.run_post_start_script():
Jul 13 14:39:42 ip-172-31-31-243 scylla_post_start.py[7853]:   File "/opt/scylladb/scylla-machine-image/libexec/scylla_post_start.py", line 56, in run_post_start_script
Jul 13 14:39:42 ip-172-31-31-243 scylla_post_start.py[7853]:     post_start_script = self.instance_user_data.get("post_start_script")
Jul 13 14:39:42 ip-172-31-31-243 scylla_post_start.py[7853]: AttributeError: 'NoneType' object has no attribute 'get'
Jul 13 14:39:42 ip-172-31-31-243 systemd[1]: scylla-image-post-start.service: Main process exited, code=exited, status=1/FAILURE
Jul 13 14:39:42 ip-172-31-31-243 systemd[1]: scylla-image-post-start.service: Failed with result 'exit-code'.
Jul 13 14:39:42 ip-172-31-31-243 systemd[1]: Failed to start Scylla cloud image post start service.
```

**Comment by syuu1228** [comment]
I think this is duplicated with #270

**Comment by yaronkaikov** [comment]
@fruch I think you fixed this in https://github.com/scylladb/scylla-machine-image/pull/378 , right?

**Comment by fruch** [comment]
> @fruch I think you fixed this in https://github.com/scylladb/scylla-machine-image/pull/378 , right? 

Yes I did

**Comment by yaronkaikov** [comment]
@tarzanek This has already been fixed and it's part of 5.2/2023.1. Do we have a customer that is waiting for this?

**Comment by yaronkaikov** [comment]
> > @fruch I think you fixed this in #378 , right?
> 
> Yes I did

Closing
